build/common.sh

```diff
@@ -89,16 +89,17 @@ readonly KUBE_CONTAINER_RSYNC_PORT=8730
 # $1 - server architecture
 kube::build::get_docker_wrapped_binaries() {
   local arch=$1
+  local base_image_registry=${BASE_IMAGE_REGISTRY:-k8s.gcr.io}
```

Comment:
- Let's move this to the `Constants` section above (line 39 or so)
- Let's use just a `REGISTRY` like we do elsewhere

Reply:
will update, thx for the suggestion.

Reply:
@dims can i propose to use both `BASE_IMAGE_REGISTRY` and `REGISTRY`? 
the reason is there is two type of images during build:
base images: consumed by build process to create released image
released images: the output container images (will tag with registry and name by default), for example: kube-api-server:###, tag with k8s.gcr.io/kube-api-server:###.

the default registry for those type of images is `k8s.gcr.io`, but they can be configured with value if needed, they are different in downstream case.

Reply:
Got it. +1 @figo let's use both. default both of them to k8s.gcr.io i suppose?

Reply:
correct, default behavior does not change, they default to `k8s.gcr.io`